openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/dashboardService/DashboardServiceTestPage.java

```diff
@@ -104,10 +104,10 @@ public void addDashboardService() throws InterruptedException {
   public void editDashboardService() throws InterruptedException {
     openDashboardServicePage();
     Thread.sleep(2000);
-    wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[data-testid='searchbar']")));
-    Events.sendKeys(webDriver, By.cssSelector("[data-testid='searchbar']"), serviceName);
-    Thread.sleep(2000);
-    Events.click(webDriver, By.cssSelector("[data-testid='edit-service']"));
+//    wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[data-testid='searchbar']")));
+//    Events.sendKeys(webDriver, By.cssSelector("[data-testid='searchbar']"), serviceName);
+//    Thread.sleep(2000);
```

Comment:
Let's remove the commented code from here also in other files in the PR

Reply:
Addressed in the latest commit